src/ocl/fusionopbiasbnactivocl.cpp

```diff
@@ -8,7 +8,8 @@ namespace fusion {
 bool IsWinograd(const std::vector<solver::AnySolver>& ss)
 {
     assert(ss.size() == 1);
-    return ss[0].GetSolverDbId() == "ConvBinWinogradRxSFused";
+    auto solverId = ss[0].GetSolverDbId();
+    return (solverId == "ConvBinWinogradRxSFused" || solverId == "ConvBinWinogradRxSf2x3g1Fused");
```

Comment:
Relying on kernel name looks too hacky. @atamazov what is your opinion?

Reply:
Inevitable for now.